### PR TITLE
[gaps] new chart

### PR DESCRIPTION
**Pull request by onedr0p**
**Description of the change**

New chart for Gaps

https://github.com/JasonHHouse/gaps

**Benefits**

**Possible drawbacks**

**Applicable issues**

- fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/gaps/values.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Default values for Gaps.
+
+image:
+  repository: housewrecker/gaps
+  pullPolicy: IfNotPresent
+  tag: latest
```
Comment (bjw-s) [inline charts/gaps/values.yaml:6]: Are there no versioned images for this? 🥺

Comment (onedr0p) [reply charts/gaps/values.yaml:6]: On my list to create a image. Maintainer doesn't follow semver well and nit multiarch images

Comment (onedr0p) [reply charts/gaps/values.yaml:6]: But we can use this as is for now until i get to that

Comment (bjw-s) [reply charts/gaps/values.yaml:6]: Cool!

**Review by bjw-s — APPROVED** [review]

**Pull request merged**